R/plotEGRET.R

```diff
@@ -0,0 +1,51 @@
+#' Create an EGRET-style plot
+#'  
+#' Use a loadflex object, but create an EGRET plot.
+#' 
+#' @param plot.name the name of the plot the user wants to create. Current
+#' options: ConcTime, ConcQ, FluxTimeDaily, and FluxQ. Each require different
+#' data, see Details.
+#' @param intdat data.frame of interpolation data
+#' @param estdat data.frame of estimation data
+#' @param preds data.frame of load predictions
+#' @param meta loadflex metadata object; it must include constituent,
+#' flow, conc.units, custom (station abbreviation: sta.abbr, and a short
+#' name for the constituent: consti.name)
+#' @param preds.type character specifying if the predictions being used are
+#' concentrations ("Conc") or fluxes ("Flux").
+#' @param ... additional arguments to pass to the plot
+#'
+#' @details ConcTime, ConcQ, and FluxQ require \code{intdat, meta}. FluxTimeDaily
+#' requires \code{intdat, estdat, preds, meta}.
+#'
+#' @importFrom EGRET plotConcTime
+#' @importFrom EGRET plotConcQ
+#' @importFrom EGRET plotFluxTimeDaily
+#' @importFrom EGRET plotFluxQ
+#'
+#' @export
+plotEGRET <- function(plot.name, intdat = NULL, estdat = NULL, preds = NULL, 
```

Comment:
Drawbacks to forcing users to load both loadflex and EGRET just to make plots:
1. their experience is less unified right away - they need to load and know about two different packages.
2. We need some way to let users know when a plot isn't appropriate for a given model - in particular, only some plots will be relevant to interpolation and composite models, and we'd like to make some plots specifically for composite models.

The benefits are:
1. the help files thing laura just mentioned. users will benefit from knowing about the customization options.
2. making sure EGRET gets credit for the pretty & thoughtful plots. we could take care of credit in ways other than naming a function after EGRET.
3. we wouldn't need to change loadflex to adapt if the EGRET interface changed, and less code in loadflex = less to maintain. 

We probably need to talk more about drawback # 2 - i'm not sure yet what the answer is here.